### DISCUSSION TITLE
Import from CSV

### DISCUSSION THREAD

**Discussion by learyjk** [planetscale/discussion#324] [Q&A] [answered]
Hello,

I created a db and table with schema I need. Is there an easy way to load data from CSV? 

I tried:

`LOAD DATA INFILE 'my-file.csv' into table my_table fields terminated by ',' ignore 1 rows;`

I get error:

`The MySQL server is running with the --secure-file-priv option so it cannot execute this statement (errno 1290)`

I found some stuff on Stack Overflow but it was also for local installs of mysql so I'm a little lost now.

Thank you!

**Comment by orware** [comment, marked as answer]
Hi @learyjk,

The way our platform is setup this is part of our [MySQL Compatibility Limitations](https://planetscale.com/docs/reference/mysql-compatibility) so the option to use `LOAD DATA INFILE` is unavailable.

The easiest alternative would be to create a script in your favorite programming language, that can read your CSV file and run the needed `INSERT` queries to get the data into your PlanetScale database.

Depending on the size and sensitivity needs of your data, there may be snippets of code or tools available online where you can add your CSV file and get back a file where each line has been converted into an `INSERT` query, which would allow you to connect via our `pscale shell` [command](https://planetscale.com/docs/reference/shell) and run a `SOURCE insert-queries.sql` type command to import the data in that manner.

Hopefully this helps provide some additional guidance! Please keep us posted on how things go or you can also [create a support ticket](https://support.planetscale.com/) with us to share more information you may not be able to directly include here publicly.

**Reply by learyjk** [reply]
Thanks @orware ! I might explore just uploading it to another mysql db and then importing it using your import tool. Are there any recommended databases that work well with your import tool?

**Reply by orware** [reply]
@learyjk,

Sorry for the delay, I only just came across your reply above! If you use a MySQL 8 database that should be the most compatible with PlanetScale!
